griffon/services/core_queries.py

```diff
@@ -286,11 +292,14 @@ def __init__(self, params: dict) -> None:
         self.search_related_url = self.params.get("search_related_url")
         self.search_redhat = self.params.get("search_redhat")
         self.search_community = self.params.get("search_community")
-        self.search_upstreams = self.params.get("search_upstreams")
+        self.search_all_upstreams = self.params.get("search_all_upstreams")
         self.filter_rh_naming = self.params.get("filter_rh_naming")
         self.no_community = self.params.get("no_community")
+        self.search_provides = self.params.get("search_provides")
+        self.search_upstreams = self.params.get("search_upstreams")
         if not self.no_community:
             self.community_session = CommunityComponentService.create_session()
+        self.include_inactive_product_streams = self.params.get("include_inactive_product_streams")
```

Comment:
Hmmm, we had this option but didn't use it?

Reply:
we used it in the operation and output filtering, never in the service as we did not have ?active_streams=True yet in component registry (which we do now)